
OptionalInt32 max_time_ms
OptionalLocalizationMovement mvmt_type
